app/src/components/main/paginationToolbar/noItemMessage/noItemMessage.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { defineMessages, injectIntl } from 'react-intl';
+import Parser from 'html-react-parser';
+import classNames from 'classnames/bind';
+import ErrorIcon from 'common/img/error-inline.svg';
+import styles from './noItemMessage.scss';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
+  noResults: {
+    id: 'LogsGrid.noResults',
+    defaultMessage: 'No results found',
+  },
+});
+
+export const NoItemMessage = injectIntl(({ intl }) => (
```

Comment:
Can you use existed component for this from `components/main/noItemMessage`?

Reply:
Yes, fixed